src/functionaltests/java/com/ericsson/ei/templates/TemplatesTestSteps.java

```diff
@@ -69,8 +69,7 @@ public void list_available_files() throws Exception {
         assertEquals(HttpStatus.OK, response.getStatusCode());
 
         String actualSubscriptionsValue = objMapper.readValue(response.getBody(), JsonNode.class).get("subscriptions").asText();
-        assertEquals("List all files did not return expected subscriptions file value. "
-                + "\nExpected: " + expectedSubscriptionsValue + "\nActual: " + actualSubscriptionsValue,
+        assertEquals("List all files did not return expected subscriptions file value. ",
```

Comment:
Use of Hamcrest framework would have made tests more readable.... But not necessarily part of this pull request.